### ISSUE TITLE
Model Evaluation not working

### ISSUE THREAD

**Issue by filippobartolucci** [vishal3477/pro_loc#2]
Hi there,

I'm encountering an issue while trying to use the provided evaluation code with your models. Specifically, the detection model consistently outputs the same value for all images, regardless of whether they are real or fake. I've observed that the model functions correctly when used in training mode, but that's not the intended approach for evaluation.

I followed the instructions outlined in the repository, using the loc_det_model.pickle model and the CelebA dataset.

Any ideas on how to solve this problem?
Thank you for your assistance.

Here is an example of an output of the model in eval mode

```
tensor([[6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01],
        [6.5181e-05, 9.9995e-01]], device='cuda:0', grad_fn=<SigmoidBackward0>)
```
Among the 8 images in the batch, 4 are real, and the remaining 4 are manipulated. However, the output is the same for all of them

Python: 3.7.12
Torch: 1.13.1
Tensorflow: 1.13.1
